dev/Common/TelemetryHelper.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+#pragma once
+class TelemetryHelper
+{
+public:
+    TelemetryHelper()
+    {
+        m_isPackagedApp = AppModel::Identity::IsPackagedProcess();
```

Comment:
It seems like both of these fields can be marked `const` and initialized in the initializer list.

Reply:
Updated!